homeassistant/components/bang_olufsen/strings.json

```diff
@@ -25,6 +25,137 @@
       }
     }
   },
+  "device_automation": {
+    "trigger_type": {
```

Comment:
If you decide to still support device triggers, I'd suggest to have one type for each kind of keypress, and a subtype for each button. That should avoid overwhelming the user with a huge list of triggers.

Example: https://github.com/home-assistant/core/blob/43642a8eedcfdf5adf767e7da0e19eb52d9143b6/homeassistant/components/hue/strings.json#L43-L75